src/main/java/com/example/MovieApplication/MovieRating.java

```diff
@@ -0,0 +1,85 @@
+package com.example.MovieApplication;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+@Entity
+public class MovieRating {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+
```

Comment:
Java has a good feature to validate fields with annotation. It may be helpful to minimize code.
@NotNull,  @Min(1),  @Max(5), and etc